src/Fetch.mli

```diff
@@ -212,7 +212,20 @@ module Response : sig
   external text : string Js.Promise.t = "" [@@bs.send.pipe: t]
 end
 
+module FormData : sig 
+  type t = formData
+
+  external make : unit -> t = "FormData" [@@bs.new]
+
+  external mutableAppendObject : string -> < .. > Js.t -> unit = "append" [@@bs.send.pipe : t]
+
+  external mutableAppendString : string -> string -> unit = "append" [@@bs.send.pipe : t]
+  external mutableDelete : string -> unit = "delete" [@@bs.send.pipe : t]
+  external mutableSetObject : string -> < .. > Js.t -> unit = "set" [@@bs.send.pipe : t]
+  external mutableSetString : string -> string -> unit = "set" [@@bs.send.pipe : t]
```

Comment:
These don't really need to be prefixed with `mutable`, I think, since returning `unit` should be a good enough indication that they're mutating.